django/contrib/admin/templates/admin/actions.html

```diff
@@ -13,9 +13,9 @@
         {% if cl.result_count != cl.result_list|length %}
         <span class="all hidden">{{ selection_note_all }}</span>
         <span class="question hidden">
-            <a href="#" title="{% translate "Click here to select the objects across all pages" %}">{% blocktranslate with cl.result_count as total_count %}Select all {{ total_count }} {{ module_name }}{% endblocktranslate %}</a>
+            <a role="button" href="#" title="{% translate "Click here to select the objects across all pages" %}">{% blocktranslate with cl.result_count as total_count %}Select all {{ total_count }} {{ module_name }}{% endblocktranslate %}</a>
```

Comment:
I suspect we probably shouldn't say "click here to" as not everyone is clicking and most buttons don't say "click here to do thing", they just say "do thing" but as it's already like this and will mean translations need to change I don't think we should block this on it.

Reply:
I agree it's not good UX. I think I'd prefer if the select all box was a dropdown rather than this linky button thing

Something like this:
![image](https://github.com/user-attachments/assets/ace7eab8-a69c-4010-b800-af381a3c8dab)

If we did do this we should have a new ticket (agree doesn't block this) and I'm also not sure how we make general UI/UX design decisions

Reply:
> I'm also not sure how we make general UI/UX design decisions

I feel like it's a bit 🫠 best effort at the moment, we don't really have any UX / UI expert contributors these days.

I am trying to upskill here, so maybe some day...